js-miniapp-sample/src/pages/landing.js

```diff
@@ -13,6 +13,16 @@ type LandingProps = {
   getHostInfo: Function,
 };
 
+window.addEventListener('miniAppWebviewClosed', function (e) {
```

Comment:
Maybe we should have a separate screen in the sample mini app for testing the events? Currently the only way to open an external webview is on the "Uri Schemes" screen.

For example, the "event listener" screen could have:
- Button to open a page in the external webview
- Counter showing how many times it has received the `miniappwebviewclosed ` event 
- Counter showing how many times received `miniapppause` event
- Counter showing how many times received `miniappresume` event

What do you think?